roles/infra/roles/storages/tasks/main.yml

```diff
@@ -91,7 +91,7 @@
     username: "{{ storages[item.0.name][item.0.storage.type].username | default(omit) }}"
     password: "{{ storages[item.0.name][item.0.storage.type].password | default(omit) }}"
     port: "{{ storages[item.0.name][item.0.storage.type].port | default(omit) }}"
-    target: "{{ storages[item.0.name][item.0.storage.type].target | default(omit) }}"
+    target: "{% if storages[item.0.name][item.0.storage.type].target not in item.0.storage.volume_group.logical_units | map(attribute='target') | list %}{{ storages[item.0.name][item.0.storage.type].target | default(omit) }}{% else %}{{ omit }}{% endif %}"
```

Comment:
1) Very long line. Is it possible to break to a few lines, so "else" & "endif" will be at separate lines and indented?
2) Maybe swap the condition: "not in" --> "in". IMHO will be more readable.
3) I'm not a big expert in Ansible, so sorry if it's a stupid question. But if I understand correctly, "default(omit)" means the "item.0.storage.type].target" is optional. What will happen with the "if storages[item.0.name][item.0.storage.type].target" check then (if it doesn't exist)? Is there a potential crash (similar to Null Pointer Exception in Java)?